draft-dns-zone-digest.xml

```diff
@@ -697,9 +697,9 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
       </section>
     </section>
 
-    <section title="Verifying Zone Message Digest" anchor="verifying">
+    <section title="Verifying Zone Digest" anchor="verifying">
       <t>
-        The recipient of a zone that has a message digest record can verify the zone
+        The recipient of a zone that has a &RRNAME; RR can verify the zone
```

Comment:
Is there a formatting issue here?

Reply:
Not a formatting issue.  &RRNAME; is an XML macro that is currently set to "ZONEMD" so in the output document it would appear as "...has a ZONEMD RR ..."